src/data-fetching/api/PrezlyApi.ts

```diff
@@ -43,6 +43,11 @@ interface GetStoriesOptions {
     pinning?: boolean;
     include?: (keyof Story.ExtraFields)[];
     localeCode?: string;
+    /**
+     * When set to `true`, the result for the first page will include one extra story to place as highlighted story.
+     * This will offset each subsequent page by 1 story to account for that.
+     */
+    useHighlightedStory?: boolean;
```

Comment:
Would it make sense to rename this to `withHighlightedStory`? To match our other behavior modification flags naming, and not make it look like a React hook.

Reply:
7c8c6d80aaee6928ac8d410f107b6c630446ca10